jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2437,6 +2437,38 @@ properties:
             title: Pixel scale ratio relative to native detector scale
             type: number
             fits_keyword: PXSCLRT
+      soss_extract1d:
+        title: NIRISS SOSS Extract1D Parameters
+        type: object
+        properties:
+          width:
+            title: Aperture width used in extraction
+            type: number
+            fits_keyword: SOSS_WDTH
+          tikhonov_factor:
+            title: Tikhonov factor used in extraction
+            type: number
+            fits_keyword: TIKFAC
+          delta_x:
+            title: X-shift used in extraction
+            type: number
+            fits_keyword: SOSS_DLTX
+          delta_y:
+            title: Y-shift used in extraction
+            type: number
+            fits_keyword: SOSS_DLTY
+          theta:
+            title: Rotation angle used in extraction
+            type: number
+            fits_keyword: SOSS_ROTN
+          oversampling:
+            title: Oversampling used in extraction
+            type: number
+            fits_keyword: OVERSAMP
+          threshold:
+            title: Threshold used in extraction
+            type: number
+            fits_keyword: THRESHLD
```

Comment:
In addition to these new header keywords to store the value of extraction params, we should also add new R_xxxxxx keywords to record the names of the new ref files being used for the SOSS extraction (unless of course those are already included in the code updates being made by the NIRISS team).

Reply:
Seems unlikely, I'll add them. Didn't realize there were entries for these 😨

Reply:
Seems unlikely, it's easy to forget or miss, at least from my experience. 😬  I've added them.